src/dotnet-install.ps1

```diff
@@ -780,8 +780,15 @@ if ($isAssetInstalled) {
 New-Item -ItemType Directory -Force -Path $InstallRoot | Out-Null
 
 $installDrive = $((Get-Item $InstallRoot).PSDrive.Name);
-$diskInfo = Get-PSDrive -Name $installDrive
-if ($diskInfo.Free / 1MB -le 100) {
+$diskInfo = $null
```

Comment:
There is a way to check the space on network share as well: https://social.technet.microsoft.com/Forums/ie/en-US/80d42412-e740-4146-9ad3-70ec6755d9c3/script-to-monitor-free-space-in-unc-path-network-drive?forum=winserverpowershell

Not sure if we want to do it though.

Reply:
I would consider this a nice-to-have. I believe it doesn't meet the bar we have for maintenance mode. Let's discuss if you think otherwise.